src/main/java/com/answerdigital/answerking/controller/OrderController.java

```diff
@@ -35,7 +35,7 @@ public OrderController(final OrderService orderService) {
     }
 
     @PostMapping
-    public ResponseEntity<Order> addOrder(@Valid @RequestBody final OrderRequest orderRequest) {
+    public ResponseEntity<Order> addOrder(@Valid @RequestBody final RequestModelsOrder orderRequest) {
```

Comment:
I think OrderRequest would fit better as a name,.

Reply:
RequestModelsOrder changed to OrderRequest.